gui/src/installer/view.rs

```diff
@@ -90,6 +90,15 @@ pub fn welcome<'a>() -> Element<'a, Message> {
             .push(
                 Container::new(
                     Column::new()
+                        .push_maybe(path.as_ref().map(|p| {
+                            Row::new()
+                                .push(Space::with_width(Length::Fixed(30.0)))
+                                .push(Container::new(
+                                    button::secondary(None, " Back ")
```

Comment:
Perhaps the Back button could be labelled "Previous" to be consistent with other steps in the installer?

Reply:
Yes!